docs/02.API-REFERENCE.md

```diff
@@ -10284,106 +10286,6 @@ main (void)
 - [jerry_parse_and_save_snapshot](#jerry_parse_and_save_snapshot)
 
 
-## jerry_load_function_snapshot
-
-**Summary**
-
-Load the selected snapshot function from the specified buffer as a function object.
-
-The lexical environment of the loaded function is always the global lexical environment.
-
-*Notes*:
-- Returned value must be freed with [jerry_release_value](#jerry_release_value) when it
-  is no longer needed.
-- This API depends on a build option (`JERRY_SNAPSHOT_EXEC`) and can be checked in runtime with
-  the `JERRY_FEATURE_SNAPSHOT_EXEC` feature enum value, see [jerry_is_feature_enabled](#jerry_is_feature_enabled).
-  If the feature is not enabled the function will return an error.
-
-**Prototype**
-
-```c
-jerry_value_t
-jerry_load_function_snapshot (const uint32_t *snapshot_p,
-                              size_t snapshot_size,
-                              size_t func_index,
-                              uint32_t exec_snapshot_opts);
-```
-
-- `snapshot_p` - pointer to snapshot.
-- `snapshot_size` - size of snapshot in bytes.
-- `func_index` - index of function to load from the snapshot.
-- `exec_snapshot_opts` - any combination of [jerry_exec_snapshot_opts_t](#jerry_exec_snapshot_opts_t) flags.
-- return value
-  - function object built from the snapshot.
-  - thrown error, otherwise.
-
-*New in version 2.0*.
-
-**Example**
-
-[doctest]: # ()
```

Comment:
AFAIK, the example should be moved and adapted for the "jerry_exec_snapshot" method's doc.